config-linux.md

```diff
@@ -654,15 +655,17 @@ The following rules on parameters MUST be applied:
 
 * If either `l3CacheSchema` or `memBwSchema` is set, runtimes MUST write the value to the `schemata` file in the that sub-directory discussed in `closID`.
 
-* If neither `l3CacheSchema` nor `memBwSchema` is set, runtimes MUST NOT write to `schemata` files in any `resctrl` pseudo-filesystems.
+* If `schemata` field is set, runtimes MUST write the value to the `schemata` file in the that sub-directory discussed in `closID`. If also l3CacheSchema` or `memBwSchema` is set the value of `schemata` field must be written last, after the values from `l3CacheSchema` and `memBwSchema` has been written.
```

Comment:
I'll write  a longer comment, but I'm considering that we should document that if `schemata` is set, runtimes `MUST` ignore both the `l3CacheSchema` and `memBwSchema` fields, and instead to use the `schemata` field instead (and write its content "as-is" (that should probably also be documented on the `l3CacheSchema` and `memBwSchema` fields)

Reply:
This field also needs to be added to the JSON-schema; https://github.com/opencontainers/runtime-spec/blob/c0e90434df2acfc0f75e689098d865d1ece16ff8/schema/config-linux.json#L259-L278

As well as the go implementation; https://github.com/opencontainers/runtime-spec/blob/c0e90434df2acfc0f75e689098d865d1ece16ff8/specs-go/config.go#L795

Reply:
Woops 🙄 Added